### ISSUE TITLE
Issue with the font:

### ISSUE THREAD

**Issue by zackultimate** [ccd0/4chan-x#1461]
This worked for me until few minutes ago:
 Current font:

https://i.imgur.com/MRdPRw6.jpg

4chan font without noscript/old font:
https://i.imgur.com/TmWPFG3.jpg

Does anyone know how to fix this and get back the old font? For some reason 4chan x doesn't load the right font.

**Comment by zackultimate** [comment]
To add further, it's not the whole 4chan x that uses this weird font:

https://i.imgur.com/nxortgr.jpg

**Comment by zackultimate** [comment]
To further add again: Chrome works fine, it appears to be only a firefox problem. Any idea?

**Comment by ccd0** [comment]
Maybe you changed the page style (selector at the bottom of the page) or some font-related browser setting?
Does this happen without 4chan X?

**Comment by zackultimate** [comment]
^No, removing 4chan X brings the font to his right size. 
I notice ALL 4chan x writing is in san serif instead of Arial now, could it be related to it? When I remove 4chan x, it goes back to Arial.

Basically the issue is that it doesn't use Arial anymore.

**Comment by ccd0** [comment]
Do you have anything in custom CSS?

**Comment by zackultimate** [comment]
Just the default stuff:

https://i.imgur.com/wkpQ4Hv.jpg


Not that the whole this in in san-serif too. :/  It's like all aria script decide to become san-serif for this kind of stuff.

Also ON CHROME, it appears to display it on Arial.

**Comment by ccd0** [comment]
Does it happen in a [fresh profile](https://support.mozilla.org/en-US/kb/profile-manager-create-and-remove-firefox-profiles)?

**Comment by zackultimate** [comment]
Yes. Also with greeasemoney alternative and a fresh firefox install.

I'm starting to think Firefox is replacing all "arial" font of that kind with san-serif?

**Comment by zackultimate** [comment]
Small update:

"Pressing original form" still gives me the old form. 

[img]https://i.imgur.com/YcrC4Fb.jpg[/img]

**Comment by ccd0** [comment]
https://boards.4chan.org/qa/thread/1512249#p1512397
Fixed now?

**Comment by zackultimate** [comment]
^That wasn't me, but yes. If your PC is set on JP because you play VN, etc you will get the "bug". It only does it on FF55+.

In order to fix it you need to switch back to EU/US, but this means I won't be able to boot VN. :/

**Comment by ccd0** [comment]
Can't you use [Locale Emulator](https://github.com/xupefei/Locale-Emulator) for that?

If I can reproduce it, I'll look into why it only happens with 4chan X on.

**Comment by OrangeFlavored** [comment]
I think all you have to reproduce is switch to Japanese locale and have latest version of FF?  Happened to me too starting a few days ago when the update finally applied itself, much harder to read now.

**Comment by guft** [comment]
I'm having this issue as well. I have a Japanese locale on this machine so I can play JP locale games, and after updating to FF v55 4chan X's font got all weird and replaced my \ key with yen signs:
https://i.imgur.com/F7ieZKY.jpg

My language and keyboard inputs are defaulted to English even though the locale is Japanese, and my language bar is also currently set to English, the issue only started after Firefox updated.
